charts/rancher-istio/1.4.3/questions.yaml

```diff
@@ -1,3 +1,4 @@
 labels:
   rancher.istio.v1.4.3: 1.4.3
 rancher_min_version: 2.3.4-rc1
+rancher_max_version: 2.3.4
```

Comment:
looking at this, let's match this with the rancher_min_version to 2.3.4-rc1

Reply:
Will do. But will it mess with current installs of 1.4.3 on 2.3.4 ? Thought it showed an error about being invalid version or something.